pages/favorites/index.js

```diff
@@ -0,0 +1,44 @@
+import styled from "styled-components";
+import SpotCard from "../../components/SpotCard.js";
+
+export default function FavoriteSpots({ spots, setSpots }) {
+  function toggleBookmark(id) {
+    const newSpotArray = spots?.map((spot) => {
+      if (spot.id === id) {
+        return {
+          ...spot,
+          isBookmark: !spot.Bookmark,
+        };
+      } else {
+        return spot;
+      }
+    });
+    setSpots(newSpotArray);
+  }
+
+  return (
+    <>
+      <StyledBookmarkSection>
+        <ul>
+          {spots?.map((spot) => {
+            if (spot.isBookmark === true) {
+              return (
+                <SpotCard
+                  spot={spot}
+                  key={spot.name}
+                  toggleBookmark={toggleBookmark}
+                />
+              );
+            } else {
+              return null;
+            }
+          })}
```

Comment:
Currently nothing is displayed in your favorites page. The problem is that you store your spots in your `index.js` and not in your `_app.js`. You need to list up the state. :)